runtime/redirector/module.xml

```diff
@@ -165,45 +165,6 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		</libraries>
 	</artifact>
 
-	<artifact type="shared" name="jvm_jdk12" scope="redirector" loadgroup="top" buildlocal="true" appendrelease="false">
```

Comment:
Why is this no longer needed? (the answer should be in the commit comment)

Reply:
Commit message has been improved.